### DISCUSSION TITLE
Prepending options to commands when using git directly

### DISCUSSION THREAD

**Discussion by ibagov** [gitpython-developers/GitPython#1520] [Q&A] [answered]
Is it possible to prepend options to commands when using git directly? I want to prepend the "-c" option to the `repo.git.commit()` command as in https://stackoverflow.com/a/19841177 but cannot find a solution so far.

**Comment by Byron** [comment, marked as answer]
I don't think this is possible yet. As alternative, it should be possible to pass any amount of configuration flags [via environment variables](https://git-scm.com/docs/git-config#Documentation/git-config.txt-GITCONFIGCOUNT).
